osmaxx/utilities/shortcuts.py

```diff
@@ -32,7 +32,7 @@ def inform_mail(self, subject, mail_body, warn_if_no_email=True):
         try:
             email_address = self.recipient.email
             mail.send_mail(
-                '[OSMAXX] ' + subject,
+                '[OSMaxx] ' + subject,
```

Comment:
I'd lean towards using the setting here. Then, if we ever use [`mail_admins()`](https://docs.djangoproject.com/en/1.9/topics/email/#mail-admins) or [`mail_managers()`](https://docs.djangoproject.com/en/1.9/topics/email/#mail-managers), (where Django would make use of the setting automatically), the setting is already in place.